### DISCUSSION TITLE
Victoria BC CANADA

### DISCUSSION THREAD

**Discussion by aiotabloc** [OpenMined/courses#190] [Introduce Yourself]
Hello everyone.......This is a very interesting time and I am keen to read all of the various perceptions of what is happening in the world today.

**Comment by yemikifouly** [comment]
Hello! 👋 Welcome to the discussion board! Great to have you here! 🙂 These are indeed very interesting times to tackle topics around privacy. Have you started the course? How are you liking it?

**Comment by aiotabloc** [comment]
Hello Yemissi B. Kifouly,

 

I have begun the course and currently at lesson 4: Technical Tools for Output Privacy.  I have found what I have learned so far quite interesting…..privacy and transparency.

 

It was easier to understand your course so far because of the material I had studied for the Bitcoin professional certification exam I took a few weeks ago.

 

Thanks,

Chris
